OrbitGl/ProcessManager.h

```diff
@@ -39,8 +41,11 @@ class ProcessManager {
   virtual void SetProcessListUpdateListener(
       const std::function<void(ProcessManager*)>& listener) = 0;
 
+  virtual outcome::result<std::vector<ModuleInfo>, std::string> GetModuleList(
+      uint32_t pid) = 0;
+
   // Get a copy of process list.
-  virtual std::vector<ProcessInfo> GetProcessList() const = 0;
+  virtual std::vector<ProcessInfo> process_list() const = 0;
```

Comment:
Why this change?

Reply:
I wanted to distinguish between calls that result in client-server communication and calls that just get stored data from object itself. this is probably not ideal because it is kind of implicit. I will probably change it..

Reply:
okay